util/grpc/grpc.go

```diff
@@ -157,6 +158,7 @@ func WithTimeout(duration time.Duration) grpc.UnaryClientInterceptor {
 		clientDeadline := time.Now().Add(duration)
 		ctx, cancel := context.WithDeadline(ctx, clientDeadline)
 		defer cancel()
+		fmt.Printf("calling %s with timeout %v from now\n", method, duration)
```

Comment:
This is for debugging, right? please use `log.Debug` instead.

Reply:
Ah I'll just kill that line. I was trying to figure out who was calling what, and `log` wasn't available in that module.

(I assume "`log` wasn't available in that module" is a reasonable thing to say. Basically GoLand complained, so I went with easy. :-))